docs/devices/014G2461.md

```diff
@@ -40,6 +44,9 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 The minimal value is `0` and the maximum value is `100`.
 The unit of this value is `%`.
 
+### Mount mode (boolean)
```

Comment:
On which property is this exposes? I cannot find anything about e.g. `mount_mode`.

Reply:
Nevermind, it's `mounted_mode`.